feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/radius_proxy.c

```diff
@@ -361,6 +361,95 @@ static bool radius_proxy_config_set(struct schema_Radius_Proxy_Config *conf)
 	return true;
 }
 
```

Comment:
Is it not possible to check for "*" realm in radius_proxy_config_set itself in. "/* Configure realm block */" and add it at the end instead of deleting it and re-adding.

Reply:
The radius_proxy_config_set function is called per ovsdb row insert event in the Radius_Proxy_Config OVSDB table.  I can't tell if the ovsdb row with the wildcard is the last row when the radius_proxy_config_set function is called.